config/application.rb

```diff
@@ -9,5 +9,13 @@
 module AskdarcelApi
   class Application < Rails::Application
     config.api_only = true
+
+    # Algolia
+    config.x.algolia.application_id = ENV['ALGOLIA_APPLICATION_ID']
+    config.x.algolia.api_key = ENV['ALGOLIA_API_KEY']
```

Comment:
https://github.com/ShelterTechSF/cloudformation/pull/23

The template will have placeholders in repo but secrets will be set in AWS